src/NuGet.Core/NuGet.Packaging/Signing/Verification/SignatureVerificationStatusFlags.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Packaging.Signing
+{
+    [Flags]
```

Comment:
When you use `FlagsAttribute`, the member values should be powers of 2.  Instead, you're relying on default values assignment which are automatically incrementing by 1 over the previous value.  This means you can't have a flags value which is a combination of any 2 or more flag values.  This may be intentional; if so, just remove the `[Flags]` attribute decoration.

See [this](https://msdn.microsoft.com/en-us/library/system.flagsattribute(v=vs.110).aspx#Anchor_6).